build.gradle

```diff
@@ -23,15 +23,15 @@ ext {
     earliestSupportedMajor: '2020.1',
     // Most recent minor versions of all major releases
     // between earliest supported (incl.) and latest stable (excl.), used for binary compatibility checks
-    latestMinorsOfOldSupportedMajors: ['2020.1.4', '2020.2.4'],
-    latestStable: '2020.3.3',
+    latestMinorsOfOldSupportedMajors: ['2020.1.4', '2020.2.4', '2020.3.3'],
+    latestStable: '2021.1',
     // Note that we have to use a "fixed snapshot" version X.Y.Z-EAP-SNAPSHOT (e.g. 211.4961.33-EAP-SNAPSHOT)
     // rather than a "rolling snapshot" X-EAP-SNAPSHOT (e.g. 211-EAP-SNAPSHOT)
     // to ensure that the builds are reproducible.
     // EAP-CANDIDATE-SNAPSHOTs can be used for binary compatibility checks,
     // but for some reason aren't resolved in UI tests.
     // Generally, see https://www.jetbrains.com/intellij-repository/snapshots/ -> Ctrl+F idea
-    eapOfLatestSupportedMajor: '211.6693.65-EAP-SNAPSHOT'
+    eapOfLatestSupportedMajor: null
```

Comment:
?

Reply:
We don't have any EAP to test against currently... until a one is released for 2021.2